index.html

```diff
@@ -163,6 +166,94 @@ <h1 class="clr-blue">About me</h1>
       </p>
       <button class="btn btn-2">Get my resume</button>
     </section>
-    <section></section>
+    <section class="skills center">
+      <div class="skill">
+        <img src="imgs/p1.png" alt="skills" class="p1" />
+        <h1>Languages</h1>
+        <ul class="li">
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">HTML</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">CSS</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Javascript</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Ruby</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+      </div>
+    </section>
+    <section class="skills center frameworks">
+      <div class="skill">
+        <img src="imgs/p2.png" alt="skills" class="p1" />
+        <h1>Frameworks</h1>
+        <ul class="li li-3">
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">React.js</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Ruby</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">RSpec</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+        <ul class="li li-3">
+          <li class="li-2">Capybara</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Selenium</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+      </div>
+    </section>
+    <section class="skills center">
+      <div class="skill">
+        <img src="imgs/p3.png" alt="skills" class="p1" />
+        <h1>Skills</h1>
+        <ul class="li">
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Database Management</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Version Control</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+        <ul class="li">
+          <li class="li-2">CLI</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Web Development</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">API Design</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+      </div>
+    </section>
```

Comment:
As specified by the previous reviewer please do as he said.  Read his comment below 👇 

You're doing a good job 👏 separating each category with a section element but please make sure to use `ul` and `li` for categories in the "About myself" section. "Languages", "Frameworks", and "Skills" should be list items, as specified in the requirements.
So instead of having separate section element for each category replace it with the `li` tag and place them in a `ul` tag